cont3xt/vueapp/src/components/integrations/IntegrationTable.vue

```diff
@@ -250,37 +254,88 @@ export default {
 
       if (!newSearchTerm) {
         this.filteredData = this.data;
+        this.highlightData = null;
         this.setTableLen();
         syncWithParent();
         return;
       }
-      this.filteredData = this.data.filter((row) => {
-        let match = false;
-        const query = newSearchTerm.toLowerCase();
 
-        for (const field of this.fields) {
+      /**
+       * @param arr the array to be simultaneously filtered/mapped over
+       * @param filterMapFunc (arrElement) -> [boolean, outputElement]
+       */
+      const filterMap = (arr, filterMapFunc) => {
```

Comment:
I've seen folks recommend using reduce() as a solution for filter+map, did you look at that?

Reply:
Yea, I noticed that (and meant to refactor it to do so, but I think I got side-tracked somewhere along the way)—I switched to using reduce in my latest commit.